src/libcore/num/mod.rs

```diff
@@ -212,11 +212,20 @@ pub mod diy_float;
 
 mod wrapping;
 
+macro_rules! usize_isize_to_xe_bytes_doc {
+    () => {"
+
+This function returns an array of length 2, 4 or 8 bytes depending
+on target pointer size.
```

Comment:
While I'm not a native English speaker, I feel like there should be a `the` before `target`.